src/openrct2-ui/windows/TitleExit.cpp

```diff
@@ -43,38 +64,11 @@ rct_window* WindowTitleExitOpen()
 {
     rct_window* window;
```

Comment:
Make the declaration and definition part of the same statement, you can even return directly, as the only statement that will remain is the definition and the return.
```cpp
    return WindowCreate<TitleExitWindow>(
        WindowClass::TitleExit, ScreenCoordsXY(context_get_width() - 40, context_get_height() - 64), 40, 64,
        WF_STICK_TO_BACK | WF_TRANSPARENT);
```